src/node-lib/curriculum-api-2023/index.ts

```diff
@@ -48,22 +48,7 @@ const teachersHomePageData = z.object({
   keyStages: z.array(keyStageSchema),
 });
 
```

Comment:
Naming is tough <img src="https://media0.giphy.com/media/26gR0YFZxWbnUPtMA/giphy.gif"/>

Some of this potentially could be extrapolated to the curriculum schema lib?